$NetBSD: patch-Cargo.toml,v 1.2 2020/08/23 14:57:43 kamil Exp $

Use portaudio by default

Use libc for setlocale() to fix non-ASCII character printing.
https://github.com/hrkfdn/ncspot/issues/221

--- Cargo.toml.orig	2020-08-13 08:27:21.000000000 +0000
+++ Cargo.toml
@@ -42,17 +42,12 @@ clipboard = { version = "0.5", optional 
 url = "1.7"
 strum = "0.17.1"
 strum_macros = "0.17.1"
+libc = "0.2"
 
 [dependencies.cursive]
 version = "0.15"
 default-features = false
 
 [features]
-share_clipboard = ["clipboard"]
-alsa_backend = ["librespot-playback/alsa-backend"]
-pulseaudio_backend = ["librespot-playback/pulseaudio-backend"]
-rodio_backend = ["librespot-playback/rodio-backend"]
 portaudio_backend = ["librespot-playback/portaudio-backend"]
-termion_backend = ["cursive/termion-backend"]
-mpris = ["dbus"]
-default = ["share_clipboard", "pulseaudio_backend", "mpris", "cursive/pancurses-backend"]
+default = ["portaudio_backend", "cursive/pancurses-backend"]
